strings.txt

```diff
@@ -13816,7 +13830,7 @@ DELETE_WARNING
 	EN	Are you sure you want to delete? Check 'confirm delete' to delete your playlist.
 	ES	¿Confirma que desea eliminar? Seleccione 'Confirmar eliminación' para eliminar la lista de reproducción.
 	FI	Haluatko varmasti poistaa ? Poista soittoluettelo valitsemalla Vahvista poisto.
-	FR	Voulez-vous vraiment supprimer la liste de lecture? Cliquez sur Confirmer suppression pour supprimer la liste de lecture.
+	FR	Voulez-vous vraiment supprimer la liste de lecture ? Cliquez sur 'Confirmer suppression' pour supprimer la liste de lecture.
```

Comment:
In other places you were using double quotes `"` rather than single quotes `'`.

Reply:
You are right. It would be better to put the same thing everywhere. Until now, I have chosen to use the same character as in the English text, without asking myself any questions. It is the shyness of the beginner! ;-)

I am correcting this text ...

In a future PR, I will try to go over all the messages in French to standardize them.

Reply:
Sorry ... I'm not very good with GitHub ...

I can't edit this strings.txt file directly in the github.com editor (because the file is too big), and I don't know how to complete the pull request from the git client and my fork (I'm new to git ).

Can we leave this file as is, or can you replace single quotes with double quotes for me?

As I said before, I will try to go through all the files in the coming days to standardize this quote concern.

Reply:
No worries. Just leave it as it is. I'll merger later tonight and apply the corrections manually. You're probably right about the _DESC string. I will double check whether it's even still being used.

Reply:
BTW: quotes are language specific.... see eg.  the Norwegian version of this string. But it might be a daunting task to verify them all...